isotope-intensity-estimation.Rmd

```diff
@@ -210,70 +222,81 @@ of one or more neutrons, the difference in mass between them isn't exactly
-the "C13" case
-
-$((n_C-2,2), (n_N, 0), (n_O, 0, 0), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "2C13" case
-
-$((n_C, 0), (n_N, 0), (n_O-1, 1, 0), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "O17" case
-
-$((n_C, 0), (n_N, 0), (n_O-1, 0, 1), (n_S, 0, 0), (n_{Cl},0))$ corresponding to 
-the "O18" case
-
-...
-et cetera.
-
-For example ((5,1), (4, 0), (3, 0, 0), (2, 0, 0), (1,0)) and 
-((1,1), (2, 0), (3, 0, 0), (4, 0, 0), (5,0)) are both in the "C13" family and 
-correspond to the md: mass of C13 - mass of C12
-
-## Selecting "significant" families of configurations
-
-Here we try to determine which families of configurations (or equivalently which 
-mass differences with respect to the monoisotopic mass) to keep.
-
-A first idea could be to compute the isotope pattern for each element in HMDB  
-and store the configurations (more precisely, the families associated that 
-configuration) that are related to isotope pattern peaks with intensity higher 
-than a certain threshold (whose value we have to decide). Then we could decide 
-to keep the most frequent ones (and we would have to decide a certain cutoff).
-
-```{r}
+In a compound with $n$ atoms of element $X$ ($n_X$) each of the atoms can be in
+a specific isotope form $\ce{ ^{y}X }$ such as $\ce{^{12}C}$ or $\ce{^{13}C}$
+(with the prevalence of each isotope form for each element being known). Each
+combination of such isotopes $I_X$ (e.g. $\ce{3 ^{12}C}$ $\ce{2 ^{13}C}$ 
+$\ce{4 ^{14}C}$ will result in a different shift in mass $md_{I_X}$. 
+The probability $P(I_X)$ of observing such a combination of isotopes for an
+element $X$ can be computed using the multinomial distribution. The overall mass
+difference $md$ with respect to the monoisotopic mass is determined by the
+combination (number and variant) of isotopes of each element $X, Y,... Z$ in the
+molecule which defines the
+(isotopologue)[https://en.wikipedia.org/wiki/Isotopologue] of the compound $I =
+(I_X, I_Y... I_Z)$ and is given by the sum $md = md_{I_X}+ md_{I_Y}
+... +md_{I_Z}$. Finally the probability of observing the isotopologue $I$ (and
+the corresponding mass difference $md$) is $P(I) = P(md) = P(I_X)*
+P(I_Y)...*P(I_Z)$ i.e. the product of the probability of observing each isotope
+variant related to each element.
+
+There is a very high number $N_{I_X} = \binom{q + n_X -1}{n_X}$ of possible
+combinations of isotopes $I_X$ for the atoms of $X$ and a even greater number
+$N_{I_X}* N_{I_Y}...*N_{I_Z}$ of possible isotopologues $I=(I_X, I_Y...,
+I_Z)$. Here we keep only those that are significant for our purpose, i.e. that,
+based on the isotope pattern calculated from the chemical formulas of all
+compounds in HMDB, would result in an intensity that is higher than a specified
+proportion of the monoisotopic peak M0. These are identified in the following
+section.
+
+<!-- @andrea: you are now defining these "families" based on HMDB and no longer -->
```

Comment:
I think we don't need this whole part anymore because you're now defining the *most likely isotopologues* in the following section, correct?